client/components/aboutUs/Resources.js

```diff
@@ -0,0 +1,310 @@
+        points in their knowledge base, provide resources to improve in those
+        areas, and offer some ways to better prepare for technical interviews
+      </p>
+    </>
+  ),
+};
+
+export const ABOUT_DATA = [
+  {
+    id: 1,
+    svg: <MessageSVG />,
+    title: "What in the world is Chingu?",
+    text: (
+      <p>
+        Chingu is a developer community that strives to help people level-up by
+        building real projects in a team setting with others focused on
+        improving their skill set. The benefits extend past tech and into
+        getting a better grip on the soft-skills needed when working in a team
+        environment; things like working through project ideas, resolving merge
+        conflicts, and planning sprints. Learn more at ({chinguIOLink})
+      </p>
+    ),
+  },
+  {
+    id: 2,
+    svg: <LockSVG />,
+    title: "Why Open Source?",
+    text: (
+      <p>
+        In keeping with the theme of helping fellow developers, the project
+        itself serves as a way to improve confidence and ability when
+        contributing to open source. This project is open to contributors at all
+        skill levels, and the maintainers / community are here to help when
+        questions arise.
+      </p>
+    ),
+  },
+  {
+    id: 3,
+    svg: <LinkSVG />,
+    title: "How can I contribute?",
+    text: (
+      <p>
+        Ready to jump in and contribute? Check out our ({contributeLink}) or
+        take a look at the ({repoLink})!
+      </p>
+    ),
+  },
+  {
+    id: 4,
+    svg: <ShareSVG />,
+    title: `How should I report a bug or incorrect information?`,
+    text: (
+      <>
+        <p>
+          First of all, thanks for taking the time to report any issues you find
+          with our app! We are using GitHub issues to track any bugs or
+          incorrect information so they can be resolved by the community.
+        </p>
+        <br />
+        <p>
+          Please be sure to check through the issues to make sure the one you
+          are seeing hasn’t already been reported. If it has please feel free to
+          add comments to the conversation inside the existing issue.
+        </p>
+        <br />
+        <p>
+          If creating a new issue, please follow the issue template that is in
+          place.
+        </p>
+        <br />
+        <p>Report issues / bugs / incorrect information ({issueLink})</p>
+      </>
+    ),
+  },
+  {
+    id: 5,
+    svg: <CodeSVG />,
+    title: `What are you using to build this app?`,
+    text: (
+      <>
+        <p>Our app is split into a Front-End (client) and Back-End (server).</p>
+        <br />
+        <p>
+          - The Front-End is a NextJS app and we are using Styled Components for
+          most of our styling.
+        </p>
+        <br />
+        <p>- The Back-End is a Postgres DB hosted on an Express server.</p>
+        <br />
+        <p>- Full details can be found in our (README)</p>
+        {repoGitLink}
```

Comment:
Again here, `README` should not be in parenthesis and should be hyperlinked to `repoGitLink`. Sorry again for the confusion from the design!